gfx.Blank not defined

```
# korok.io/korok/gui
korok/gui/style.go:102:10: undefined: gfx.Blank
korok/gui/style.go:107:30: undefined: gfx.Blank
```
occured when `go get`.

--- a/gui/style.go
+++ b/gui/style.go
@@ -1,12 +1,13 @@
 package gui
 
 import (
-	"korok.io/korok/gfx/font"
 	"korok.io/korok/gfx"
+	"korok.io/korok/gfx/font"
 	"korok.io/korok/math/f32"
 )
 
 type Visibility uint8
+
 const (
 	Visible Visibility = iota
 	InVisible
@@ -21,7 +22,7 @@ type Theme struct {
 	Slider      SliderStyle
 
 	// global config..
-	Normal gfx.Color
+	Normal  gfx.Color
 	Pressed gfx.Color
 	// item 之间的空隙
 	Spacing float32
@@ -35,10 +36,10 @@ type Padding struct {
 }
 
 type TextStyle struct {
-	Font font.Font
-	Color gfx.Color
-	Size float32
-	Lines int
+	Font      font.Font
+	Color     gfx.Color
+	Size      float32
+	Lines     int
 	LineSpace float32
 }
 
@@ -60,17 +61,17 @@ func (text *TextStyle) SetSize(size float32) *TextStyle {
 type InputStyle struct {
 	Visibility
 	Color, HintColor uint32
-	Size float32
+	Size             float32
 }
 
 type ButtonStyle struct {
 	TextStyle
 	Padding
-	Background struct{
-		Normal gfx.Color
+	Background struct {
+		Normal  gfx.Color
 		Pressed gfx.Color
 	}
-	Gravity f32.Vec2
+	Gravity  f32.Vec2
 	Rounding float32
 }
 
@@ -84,11 +85,9 @@ type ImageStyle struct {
 }
 
 type CheckBoxStyle struct {
-
 }
 
 type ProgressBarStyle struct {
-
 }
 
 type SliderStyle struct {
@@ -98,28 +97,28 @@ type SliderStyle struct {
 //// 这样
 func newLightTheme() *Theme {
 	return &Theme{
-		Text:TextStyle{
-			Color:gfx.Blank,
-			Size:12,
-			LineSpace:6,
+		Text: TextStyle{
+			Color:     gfx.Black,
+			Size:      12,
+			LineSpace: 6,
 		},
-		Button:ButtonStyle{
-			TextStyle:TextStyle{Color:gfx.Blank, Size:12},
-			Padding:Padding{10, 10, 10, 10},
-			Gravity:f32.Vec2{.5, .5},
-			Rounding:5,
+		Button: ButtonStyle{
+			TextStyle: TextStyle{Color: gfx.Black, Size: 12},
+			Padding:   Padding{10, 10, 10, 10},
+			Gravity:   f32.Vec2{.5, .5},
+			Rounding:  5,
 		},
-		Image:ImageStyle{gfx.White},
-		ImageButton:ImageButtonStyle{
-			ImageStyle:ImageStyle{ Tint:gfx.White},
-			Padding:Padding{0, 0,0,0},
+		Image: ImageStyle{gfx.White},
+		ImageButton: ImageButtonStyle{
+			ImageStyle: ImageStyle{Tint: gfx.White},
+			Padding:    Padding{0, 0, 0, 0},
 		},
-		Slider:SliderStyle{
+		Slider: SliderStyle{
 			gfx.LTGray, gfx.Gray,
 		},
-		Normal: gfx.LTGray,
+		Normal:  gfx.LTGray,
 		Pressed: gfx.Gray,
-		Spacing:4,
+		Spacing: 4,
 	}
 }
 

